### PR TITLE
Gdocs import flow

### PR DESCRIPTION
**Pull request by sebbean**

### CODE REVIEW

File: app/controllers/spree/admin/import_source_files_controller.rb [hunk 1]
```diff
@@ -72,16 +74,63 @@ def show_in_google
...
+    columns.each_with_index do |c,i|
+      ws[1,i+1] = c
+    end
+
+    # # ws = resource.flat_worksheet spree_current_user.google_token
+    # SpreeImporter::Exporter.new(search: :dummy, target: :product).each_with_index do |r,y| 
+    #   CSV.parse(r).first.each_with_index do |c,x| 
+    #     ws[(y+1),(x+1)] = c
+    #   end
+    # end
```
Comment (sebbean) [inline app/controllers/spree/admin/import_source_files_controller.rb]: this is here so i can figure out how to do that selective export thing

Comment (sebbean) [inline app/controllers/spree/admin/import_source_files_controller.rb]: lemme see if i can clean this up a bit right now

File: app/controllers/spree/admin/import_source_files_controller.rb [hunk 2]
```diff
@@ -72,16 +74,63 @@ def show_in_google
...
+
+    ws.save
+
+    isf = Spree::ImportSourceFile.create spreadsheet_key: ss.key
+    # isf.import_from_google! spree_current_user.google_token
```
Comment (weeksie) [inline app/controllers/spree/admin/import_source_files_controller.rb]: We still using the commented out stuff or is it deadz?

File: app/controllers/spree/admin/import_source_files_controller.rb [hunk 3]
```diff
@@ -72,16 +74,63 @@ def show_in_google
...
     source_file.import!
     render json: true
   end
 
-  def index
-    respond_with @collection
-  end
+  # def index
+  #   respond_with @collection
+  # end
```
Comment (weeksie) [inline app/controllers/spree/admin/import_source_files_controller.rb]: Dead?

File: app/models/spree/import_source_file.rb [hunk 4]
```diff
@@ -102,11 +121,11 @@ def importer(force = false)
...
+  #   return if spreadsheet_url.nil?
+  #   parsed                    = Rack::Utils.parse_query spreadsheet_url.split("?").last
 
-    parsed                    = Rack::Utils.parse_query spreadsheet_url.split("?").last
-    key                       = parsed["key"]
-    self.spreadsheet_feed_url = "https://spreadsheets.google.com/feeds/worksheets/#{key}/private/"
-  end
+  #   ap parsed
+  #   self.spreadsheet_key =  parsed["key"]
+  # end
```
Comment (weeksie) [inline app/models/spree/import_source_file.rb]: Dead code?

File: app/controllers/spree/admin/import_source_files_controller.rb (hunk 2 above)
Comment (sebbean) [inline app/controllers/spree/admin/import_source_files_controller.rb]: i c u c me

File: app/models/spree/import_source_file.rb [hunk 5]
```diff
@@ -102,11 +125,113 @@ def importer(force = false)
...
-    key                       = parsed["key"]
-    self.spreadsheet_feed_url = "https://spreadsheets.google.com/feeds/worksheets/#{key}/private/"
-  end
+  # def set_feed_url_from_human_url
+  #   return if spreadsheet_url.nil?
+  #   parsed                    = Rack::Utils.parse_query spreadsheet_url.split("?").last
+
+  #   ap parsed
+  #   self.spreadsheet_key =  parsed["key"]
+  # end
```
Comment (weeksie) [inline app/models/spree/import_source_file.rb]: dead codez?

**Pull request merged**